models/aeif_cond_alpha.nestml

```diff
@@ -86,11 +86,11 @@ neuron aeif_cond_alpha_neuron:
 
     # Impulse to add to DG_EXC on spike arrival to evoke unit-amplitude
     # conductance excursion.
-    PSConInit_E nS/ms = 1.0 nS * e / tau_syn_ex
+    PSConInit_E nS/ms = nS * e / tau_syn_ex
```

Comment:
I agree with @heplesser and also think that `1 nS * e / tau_syn_ex` looks cleaner than `(e / tau_syn_ex ) * 1.0 nS` or similar.